_data/meltano/extractors/tap-github/singer-io.yml

```
@@ -28,7 +28,7 @@ settings:
     kind: date_iso8601
     description: Defines how far into the past to pull data for the provided repositories.
 capabilities:
-  - properties
+  - catalog
```

We discovered during the mob session that we can't always trust the --help output because it may be calling the common singer library without actually handling all declared CLI options.

So I dove into the docs at https://github.com/singer-io/tap-github and also I checked the source code. In fact, this tap does expect properties and does not appear to support catalog.


```suggestion
  - properties
```